recipes/lektor/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "Lektor" %}
+{% set version = "2.3" %}
+{% set hash_type = "sha256" %}
+{% set hash = "3b61045b79e05c25ea47cdc0de1e27404ec1f9ea1103024b642a7b7dd7575492" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: 0
+  skip: True  # [py3k]
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  entry_points:
+    - lektor = lektor.cli:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - babel
+    - click >=6.0
+    - exifread
+    - flask
+    - inifile
+    - jinja2 >=2.4
+    - mistune
+    - ndg-httpsclient
+    - nodejs
+    - pyasn1
+    - pyopenssl >=0.13
+    - python
+    - requests
+    - watchdog
+
+test:
+  imports:
+    - lektor
+
+  commands:
+    - lektor --help
+
+about:
+  home: https://getlektor.com
+  license: BSD 3-Clause
```

Comment:
Can we please ask upstream about getting the license file included in `MANIFEST.in`?